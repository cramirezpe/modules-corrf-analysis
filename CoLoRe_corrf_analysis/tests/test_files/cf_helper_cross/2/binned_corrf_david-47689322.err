...........80%..............40%.....30%.....20%.....10%...0%Using AVX kernel
In gridlink_mocks_double> Running with [nmesh_x, nmesh_y, nmesh_z]  = 9,9,4.  Time taken =   0.124 sec
countpairs_mocks_s_mu_double> gridlink seems inefficient. nmesh = (9, 9, 4); avg_np = 6.63e+03. Boosting bin refine factor - should lead to better performance
In gridlink_mocks_double> Running with [nmesh_x, nmesh_y, nmesh_z]  = 13,13,4.  Time taken =   0.177 sec
In gridlink_mocks_double> Running with [nmesh_x, nmesh_y, nmesh_z]  = 13,13,4.  Time taken =   0.218 sec
......................50%.....60%......70%.........90%.....100% done. Time taken =     4 mins    2 secs
100% done. Time taken =     4 mins    3 secs
............80%............60%.................30%......20%...10%...... Time taken =   0.217 sec
In gridlink_mocks_double> Running with [nmesh_x, nmesh_y, nmesh_z]  = 13,13,4.  Time taken =   0.087 sec
In gridlink_mocks_double> Running with [nmesh_x, nmesh_y, nmesh_z]  = 9,9,4. countpairs_mocks_s_mu_double> gridlink seems inefficient. nmesh = (9, 9, 4); avg_np = 6.63e+03. Boosting bin refine factor - should lead to better performance
 Time taken =   0.145 sec
In gridlink_mocks_double> Running with [nmesh_x, nmesh_y, nmesh_z]  = 13,13,4. 0%...............40%....50%.....70%.......90%......90%............70%..........50%.........30%....20%....... Time taken =   0.209 sec
In gridlink_mocks_double> Running with [nmesh_x, nmesh_y, nmesh_z]  = 13,13,4. In gridlink_mocks_double> Running with [nmesh_x, nmesh_y, nmesh_z]  = 9,9,4.  Time taken =   0.147 sec
countpairs_mocks_s_mu_double> gridlink seems inefficient. nmesh = (9, 9, 4); avg_np = 6.65e+03. Boosting bin refine factor - should lead to better performance
0%......10%................40%.......60%.......80%.........100% done. Time taken =     2 mins
100% done........90%..........70%......60%......50%.........30%..20%............0%In gridlink_mocks_double> Running with [nmesh_x, nmesh_y, nmesh_z]  = 13,13,4. In gridlink_mocks_double> Running with [nmesh_x, nmesh_y, nmesh_z]  = 9,9,4.  Time taken =   0.147 sec
countpairs_mocks_s_mu_double> gridlink seems inefficient. nmesh = (9, 9, 4); avg_np = 6.65e+03. Boosting bin refine factor - should lead to better performance
 Time taken =   0.215 sec
In gridlink_mocks_double> Running with [nmesh_x, nmesh_y, nmesh_z]  = 13,13,4.  Time taken =   0.159 sec
....10%............40%................80%...... Time taken =     4 mins    2 secs
