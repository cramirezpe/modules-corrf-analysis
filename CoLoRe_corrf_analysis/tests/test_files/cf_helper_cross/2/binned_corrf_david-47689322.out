verson 0.12
DEBUG:__main__:main:data:	/global/cscratch1/sd/cramirez/NBodyKit/multibias/s1/catalogues/1000/rsd.fits
data2:	/global/cscratch1/sd/cramirez/NBodyKit/multibias/s4/catalogues/1000/norsd.fits
randoms:	/global/cscratch1/sd/cramirez/NBodyKit/multibias/s1/catalogues/1000/rsd_rand.fits
out_dir:	/global/cscratch1/sd/cramirez/NBodyKit/cross_correlations/multibias/s1_s5/nside_2/rsd_norsd/0.1_200_41/0.5_0.7/1000//2
nthreads:	64
mu_max:	1.0
nmu_bins:	100
min_bin:	0.1
max_bin:	200.0
n_bins:	41
norsd:	False
zmin:	0.5
zmax:	0.7
drq_format:	True
zmin_covd:	0.01
zmax_covd:	1.5
zstep_covd:	0.005
random_downsampling:	1
pixel_mask:	[2]
nside:	2
log_level:	DEBUG
DEBUG:__main__:main:Data files:/global/cscratch1/sd/cramirez/NBodyKit/multibias/s1/catalogues/1000/rsd.fits
DEBUG:__main__:main:Data2 files:/global/cscratch1/sd/cramirez/NBodyKit/multibias/s4/catalogues/1000/norsd.fits
DEBUG:__main__:main:Random files:/global/cscratch1/sd/cramirez/NBodyKit/multibias/s1/catalogues/1000/rsd_rand.fits
INFO:__main__:main:Reading file:
	/global/cscratch1/sd/cramirez/NBodyKit/multibias/s1/catalogues/1000/rsd.fits
	/global/cscratch1/sd/cramirez/NBodyKit/multibias/s4/catalogues/1000/norsd.fits
	/global/cscratch1/sd/cramirez/NBodyKit/multibias/s1/catalogues/1000/rsd_rand.fits
DEBUG:__main__:main:Length of data cat: 428899029
DEBUG:__main__:main:Length of rand cat: 428899029
INFO:__main__:main:Reading second data cat
DEBUG:__main__:main:Length of data2 cat: 428921669
DEBUG:__main__:main:Applying pixel mask
DEBUG:__main__:main:Length of data after pixel mask: 8932310
DEBUG:__main__:main:Length of rand after pixel mask: 8933876
DEBUG:__main__:main:Length of data2 after pixel mask: 8968689
DEBUG:__main__:main:Length of data after z mask: 2148674
DEBUG:__main__:main:Length of rand after z mask: 2154292
DEBUG:__main__:main:Length of data2 after z mask: 2163525
INFO:__main__:main:Starting computation for current file:
INFO:__main__:main:Computing DD...
INFO:__main__:main:Computing DR...
INFO:__main__:main:Computing RR...
INFO:__main__:main:Computing RD...
DEBUG:__main__:main:Relative ellapsed time: 855.421719789505
